packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -1858,17 +2116,22 @@ export interface EnumOptions<Member extends string> {
 /**
  * An enumeration, Postgres Only
  *
+ * Fallback policy:
```

Comment:
Noticed it here, but also holds for the others. This will still render as 'Fallback policy: If this type...'. Maybe we want to highlight 'Fallback policy:' by either making it bold or using it as a header